general/bundler_imports.md

```diff
@@ -0,0 +1,160 @@
+# Bundler Imports
```

Comment:
Maybe this should be under the "dxp" directory instead of "general".

Reply:
:+1: